sdcm/cluster_cloud.py

```diff
@@ -236,11 +236,16 @@ def wait_for_cloud_init(self):
     def _init_port_mapping(self):
         pass
 
-    def set_keep_alive(self):
-        # TODO: Implement keep alive tagging for Scylla Cloud
-        self.log.info("Setting keep alive for node %s", self.name)
+    # For cloud clusters, the keep duration is calculated and set during cluster creation.
+    # _set_keep_alive and _set_keep_duration methods in base classes are invoked after cluster
+    # creation, during nodes init, when it is already late to modify cluster details.
+    # The basic implementations of these methods remain here for backward compatibility.
+    def _set_keep_alive(self) -> bool:
         return True
 
+    def _set_keep_duration(self, duration_in_hours: int) -> None:
+        pass
```

Comment:
we should have some docs about this backend, mention that we can  use the keep duration for now

Reply:
I think we can implement a followup to change the cluster name, i.e. there's API for it.